plugins/main/public/components/agents/fim/inventory/fim-data.tsx

```diff
@@ -0,0 +1,354 @@
+import React from 'react';
+import { formatUIDate } from '../../../../react-services/time-service';
+import {
+  EuiIconTip,
+  EuiAccordion,
+  EuiTitle,
+  EuiToolTip,
+  EuiIcon,
+  EuiCodeBlock,
+} from '@elastic/eui';
+
+function renderFileDetailsPermissions(value) {
+  return (
+    <EuiAccordion
+      id={Math.random().toString()}
+      paddingSize='none'
+      initialIsOpen={false}
+      arrowDisplay='none'
+      buttonContent={
+        <EuiTitle size='s'>
+          <h3>
+            Permissions
+            <span style={{ marginLeft: 16 }}>
+              <EuiToolTip position='top' content='Show'>
+                <EuiIcon
+                  className='euiButtonIcon euiButtonIcon--primary'
+                  type='inspect'
+                  aria-label='show'
+                />
+              </EuiToolTip>
+            </span>
+          </h3>
+        </EuiTitle>
+      }
+    >
+      <EuiCodeBlock language='json' paddingSize='l'>
+        {JSON.stringify(value, null, 2)}
+      </EuiCodeBlock>
+    </EuiAccordion>
+  );
+}
+
+function renderFileDetailsSize(value) {
+  if (isNaN(value)) {
+    return 0;
+  }
+  const b = 2;
+  if (0 === value) {
+    return '0 Bytes';
+  }
+  const c = 0 > b ? 0 : b,
+    d = Math.floor(Math.log(value) / Math.log(1024));
+  return (
+    parseFloat((value / Math.pow(1024, d)).toFixed(c)) +
+    ' ' +
+    ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'][d]
+  );
+}
+
+const dataFIM = [
+  {
+    field: 'file',
+    columns: {
+      name: 'File',
```

Comment:
In the `Registry` table, the `file` field was called as `Registry` instead of `File`. In the `Files` table, the `file` field is called `File`.